src/test/java/tests/auth/UserAuthServiceTest.java

```diff
@@ -122,4 +123,19 @@ public void userInvalidateToken() {
                 .body("httpStatus", equalTo("OK"))
                 .body("isSuccess", equalTo(true));
     }
+    @Test()
```

Comment:
Eğer anotasyonun içerisine bir şey yazmayacaksak parantezleri eklememek her zaman en doğru yöntemdir
```suggestion
    @Test
```